data/transactions/logic/resources_test.go

```diff
@@ -58,6 +58,12 @@ func TestAppSharing(t *testing.T) {
 		Sender:        basics.Address{1, 2, 3, 4},
 	}
 
+	pay1 := txntest.Txn{
+		Type:     protocol.ApplicationCallTx,
+		Sender:   basics.Address{5, 5, 5, 5},
+		Receiver: basics.Address{6, 6, 6, 6},
+	}
```

Comment:
name is `pay1` but no appid? or is it wrong txn type?

Reply:
name is `pay1` because it's used in as tx1 in the group it appears in.

Reply:
I misunderstood your main comment - yes, I also meant to change the type. I'll do that.